src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -255,6 +309,24 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
     this.fire("hass-more-info", { entityId: ev.model.entity.entity_id });
   }
 
+  entityCopyValue(ev) {
+    var action = ev.currentTarget.attributes.action.value;
+    if (action === "copyId") {
+      copy(ev.model.entity.entity_id);
+    } else if (action === "copyState") {
+      copy(ev.model.entity.state);
+    } else if (action === "copyAttributes") {
+      copy(this.attributeString(ev.model.entity).replace(/\n/g, "<br />"));
```

Comment:
Maybe we should do this in `yaml` instead of the text we show in the table?
`copy(safeDump(ev.model.entity.attributes));`

Reply:
Items replaced :)